etiqet-transport-solace/src/main/java/com/neueda/etiqet/transport/solace/client/SolaceClient.java

```diff
@@ -0,0 +1,147 @@
+package com.neueda.etiqet.transport.solace.client;
```

Comment:
Could we move all this stuff to a message broker specific client instead of a solace specific one? The same for the fixtures. The subscription to messages is something generic, the transport module should be the only specific part. Does this make sense to you?

Reply:
Makes sense, I'll create a MessageBrokerClient and pop the fixtures into MessageBrokerFixtures?